cmd/helm/installer/uninstall.go

```diff
@@ -21,13 +21,12 @@ import (
 
 	"github.com/ghodss/yaml"
 
+	"k8s.io/helm/pkg/kube"
 	"k8s.io/kubernetes/pkg/api"
 	kerrors "k8s.io/kubernetes/pkg/api/errors"
 	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
-
-	"k8s.io/helm/pkg/kube"
```

Comment:
this should be moved back. It's a local package.

Typical best practices for go imports are

1. stdlib
2. dependencies
3. local packages

`go fmt` would agree.